pkg/sqlite/load.go

```diff
@@ -268,7 +270,14 @@ func (s *SQLLoader) AddPackageChannels(manifest registry.PackageManifest) error
 			if err != nil {
 				errs = append(errs, err)
 				break
+			}	
+
+			// If we find 'replaces' in the circuit list then we've seen it already, break out
+			if _, ok := replaceCycle[replaces]; ok {
+			   return fmt.Errorf("Cycle detected, %s replaces %s", channelEntryCSVName, replaces)
```

Comment:
Instead of returning immediately, I think we should append the error to the errors list and break from the for loop. This way, we can continue loading other channels and leave the decision to fail up to the client.

Reply:
@njhale okay thanks for the feedback! I'll give it a try

Reply:
okay, done! apologies for the delay